src/evm/decoder.rs

```diff
@@ -90,11 +100,22 @@ impl TychoStreamDecoder {
             .insert(exchange.to_string(), decoder);
     }
 
+    /// Registers a filter function for a given exchange.
+    ///
+    /// The filter function is used to determine whether a given component should be included in
+    /// the state updates. This can be useful when only wanting to retrieve state updates for
+    /// components with a TVL higher than a given amount, for example.
```

Comment:
No this is not true, this are client side pool filter functions. We can use them to express more customized conditions that are currently not supported by the server, e.g. to filter out pools with certain attributes because e.g. they are currently not fully supported - or to implement a blacklist etc.